drops/deploy_ama_to_windows_arc_server.json

```diff
@@ -0,0 +1,24 @@
+{
+    "Title": "Deploy the Azure Monitor Agent to a Windows Azure Arc-enabled Server",
+    "Summary": "This Drop provides a guide to deploying the Azure Monitor Agent on a Windows Arc-enabled server.",
+    "Description": "This Jumpstart Drop provides a step-by-step guide to deploy the Azure Monitor Agent to a Windows Arc-enabled server.  It creates a Log Analytics Workspace and Data Collection rule to show typical performance counters and Windows Event Logs being collected.",
+    "Authors": [
+      {
+        "Name": "Mike Rhoads",
+        "Link": "https://www.linkedin.com/in/mike-rhoads/"
+      }
+    ],
+    "Source": "https://github.com/Azure/arc_jumpstart_drops/tree/main/tutorial_guide/deploy_ama_to_windows_arc_server",
+    "Type": "tutorial_guide",
+    "Difficulty": "Beginner",
+    "ProgrammingLanguage": [
+      "Bash"
```

Comment:
Is this bash or powrshell?

Reply:
I'm just showing the shell command to deploy the data collection rule so am left this at bash